### PR TITLE
add sooperlooper

### PR DESCRIPTION
**Pull request by magnetophon**
had to change the license from gpl to gpl1 due to travis complaining.
tried to squash my commits but failed, sorry!

### CODE REVIEW

**Comment by cillianderoiste** [comment]
Isn't it GPL2? https://github.com/essej/sooperlooper/blob/master/COPYING (I believe GPL1 is extremely rare).

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -12909,6 +12909,8 @@ let
       pysqlite = pkgs.pythonPackages.sqlite3;
   };
 
+  sooperlooper = callPackage ../applications/audio/sooperlooper { };
```
Comment (cillianderoiste) [inline pkgs/top-level/all-packages.nix]: Entries in all-packages.nix should be in alphabetical order and in the appropriate group. You can see the groups by searching for "###" perhaps Applications would be appropriate?

**Comment by magnetophon** [comment]
tried to squash my commits again, and failed. again...  :(
this is turning into a bit of a mess, sorry.

File: pkgs/applications/audio/sooperlooper/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, liblo, libxml2, jack2, libsndfile, wxGTK, libsigcxx,
+libsamplerate, rubberband, pkgconfig, ncurses
```
Comment (cillianderoiste) [inline pkgs/applications/audio/sooperlooper/default.nix]: Please put the "," at the start of the wrapped line.

File: pkgs/applications/audio/sooperlooper/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  src = fetchurl {
+    url = "http://essej.net/sooperlooper/${name}.tar.gz";
+    sha256 = "0n2gdxw1fx8nxxnpzf4sj0kp6k6zi1yq59cbz6qqzcnsnpnvszbs";
+  };
+
+  buildInputs = [
+   liblo libxml2 jack2 libsndfile wxGTK libsigcxx
+   libsamplerate rubberband pkgconfig ncurses
+
```
Comment (cillianderoiste) [inline pkgs/applications/audio/sooperlooper/default.nix]: ... and remove the extra line break, thanks!

**Pull request merged**